docs/src/Modal.doc.js

```diff
@@ -177,6 +181,49 @@ function Example(props) {
   />
 );
 
+card(
```

Comment:
@nachofassini fyi moved this example down - shouldn't be the first one that people see.